synapse/cores/common.py

```diff
@@ -257,6 +257,31 @@ def _loadCoreModel(self):
         for item in self.getTufosByProp('syn:prop'):
             self._initPropTufo(item)
 
+    def addDataModel(self, modname, modl):
+        '''
+        Store all types/forms/props from the given data model in the cortex.
+        This should be done on initialization of an empty cortex only.
+
+        Example:
+
+            modname = 'synapse.models.foo'
+            core.addDataModel(modname, s_dyndep.tryDynFunc(modname+'.getDataModel'))
```

Comment:
i dont think i'd introduce the dyndep / module complexity here.  it'd probably be better to just have a small model dict declared in the example and pass it in.